haret/src/vr/states/backup.rs

```diff
@@ -77,7 +78,13 @@ impl Backup {
     }
 
     pub fn commit(mut self, new_commit_num: u64, output: &mut Vec<Envelope<Msg>>) -> VrState {
-        for i in self.ctx.commit_num..new_commit_num {
+        if new_commit_num < self.ctx.log_start {
```

Comment:
`<=` is probably a better choice here, although either has the same effect due to the range below.